fluent-syntax/src/ast/helper.rs

```diff
@@ -5,15 +5,15 @@ use super::Comment;
 // This is a helper struct used to properly deserialize referential
 // JSON comments which are single continous String, into a vec of
 // content slices.
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Eq, Clone)]
```

Comment:
TODO (me): Review this and any other `Eq` traits after the above questions are answered.

Reply:
Just for reference: https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq